eng/testing/tests.android.targets

```diff
@@ -3,7 +3,7 @@
     <BundleTestAppTargets>$(BundleTestAppTargets);BundleTestAndroidApp</BundleTestAppTargets>
   </PropertyGroup>
 
-  <Import Project="$(MonoProjectRoot)\msbuild\common\LibraryBuilder.props" /> 
+  <Import Project="$(MonoProjectRoot)\msbuild\common\LibraryBuilder.props" />
```

Comment:
nit: there are many whitespacing changes in the PR, removing them would make it smaller